packages/sdk/src/types/Loyalties.ts

```diff
@@ -712,9 +732,11 @@ export interface LoyaltiesGetPointsExpirationResponseBody {
 	total: number
 }
 
-export interface LoyaltiesListCardTransactionsRequestQuery {
-	limit?: number
-	page?: number
+export interface LoyaltiesListCardTransactionsResponseBody {
```

Comment:
but this `LoyaltiesListCardTransactionsResponseBody` is also a duplicate interface now in this PR
the other one has additionally `more_starting_after?: string`